hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                $('#save-collection-warning').modal('show');
+            }
+            else
+            {
+                // copy row data and add to collection-candidate table
+                $('#collection-table-candidate').DataTable().row.add(removed_row_from_colletion.data()).draw(false);
+                // remove row obj
+                removed_row_from_colletion.remove().draw(false);
+            }
+            $('#save-new').show();
+        }
+
+
+        function add_collection_item()
+        {
+            $.each($(".row-selector:checked"),function( index, chkbox ) {
+                    var res_id = chkbox.id;
+                    var added_row = $('#collection-table-candidate').DataTable().row('#' + res_id);
+                    $('#collection-table').DataTable().row.add(added_row.data()).draw(false);
+                    added_row.remove().draw(false);
+            });
+            $('#save-new').show();
+        }
+
+        // revised metadata_update_ajax_submit
+        function collection_update_ajax_submit_new(form_id)
+        {
+            $alert_success = '<div class="alert alert-success" id="success-alert"> \
+                <button type="button" class="close" data-dismiss="alert">x</button> \
+                <strong>Success! </strong> \
+                Collection updated.\
+            </div>'
+            $alert_error = '<div class="alert alert-danger" id="error-alert"> \
+                <button type="button" class="close" data-dismiss="alert">x</button> \
+                <strong>Error! </strong> \
+                Collection failed to update.\
+            </div>'
+            $form=$('#' + form_id);
+            var resource_id_list = [];
+            $('#collection-table').DataTable().rows().every(function()
+            {resource_id_list.push(this.id);});
+            $.ajax({
+                type: "POST",
+                url: $form.attr('action'),
+                dataType: "json",
+                data: {"resource_id_list": resource_id_list},
+                success: function(result)
+                {
+                    /* The div contains now the updated form */
+                    json_response = result;
+                    if (json_response.status === 'success')
+                    {
+                        $(document).trigger("submit-success");
+                        $('#save-new').hide();
+                        if (json_response.hasOwnProperty('metadata_status')) {
+                            if (json_response.metadata_status !== $('#metadata-status').text()) {
+                                $('#metadata-status').text(json_response.metadata_status);
+                                if (json_response.metadata_status.toLowerCase().indexOf("insufficient") == -1) {
+                                    customAlert("<i class='glyphicon glyphicon-flag custom-alert-icon'></i><strong>Collection Status:</strong> sufficient to publish or make public", 3000);
```

Comment:
@zhiyuli I believe this message pops up every time when a collection is updated even when the resource is already public. You can add a check to only show this message when the resource is private.